src/Thrift/netcore/Thrift/Transports/Client/THttpClientTransport.cs

```diff
@@ -40,6 +40,7 @@ public class THttpClientTransport : TClientTransport
 
         private bool _isDisposed;
         private MemoryStream _outputStream = new MemoryStream();
+        private static readonly MediaTypeWithQualityHeaderValue XThriftMediaType = new MediaTypeWithQualityHeaderValue("application/x-thrift");
```

Comment:
Thanks for the fix. While x-thrift is accepted, it is better to be more specific about which encoding it is, i.e. ""application/vnd.apache.thrift.binary""

https://github.com/jaegertracing/jaeger/blob/7262bdc9e59933f15fb0802f0d3ba7fb642a8955/cmd/collector/app/http_handler.go#L39